spec/controllers/api/events_controller_spec.rb

```diff
@@ -171,6 +171,11 @@ def overridden_params(new_params)
           end.to change { UserProjectPreference.count }.from(0).to(1)
         end
 
+        it "should increment the project's classifiers count", :focus do
```

Comment:
still got a `:focus` set on here so the CI only tested those specs, just remove it.